src/Plugin/MediaEmbedDialog/Image.php

```diff
@@ -411,7 +443,7 @@ public function preRender(array $element) {
    * @return \Drupal\Core\Url
    *   Constructed url object.
    */
-  protected function getLinkObject($link_path, array $link_options = []) {
+  protected static function getLinkObject($link_path, array $link_options = []) {
```

Comment:
Does this need to be static?

Reply:
it doesn't need to be static but it doesn't use `$this` which allows it to be static. :man_shrugging: